src/components/Books.jsx

```diff
@@ -1,24 +1,24 @@
-import React from 'react'
-import Form from './Form'
-import Book from './Book'
-import Navbar from './Navbar'
-import { useSelector } from 'react-redux'
+/* eslint-disable linebreak-style */
+/* eslint-disable max-len */
```

Comment:
- [ ] [REQUIRED] Good job so far! Please fix linter errors without this kind of comment `/* eslint-disable ... */` because this type of comment is used in very few cases, and in your case, you can solve such errors yourself, and I think if implement my suggestion that this will increase your work experience and skills 😃